### PR TITLE
Improve the readability of crio.8.md

### PR DESCRIPTION
**Pull request by willstudy**
- **What I did**
Hello, I edited the `crio.8.md` again and want to improve readability using better format.
- **How I did it**
Vim
- **How to verify it**
Markdown preview
- **Description for the changelog**
Improve the readability of crio.8.md

### CODE REVIEW

**Comment by rhatdan** [comment]
@willstudy Why did you close this?

**Comment by TomSweeneyRedHat** [comment]
@willstudy Thanks for the PR!  You need to sign your commit "git commit --amend -s" and it looks like there are some whitespace errors that need tending.

File: docs/crio.8.md [hunk 1]
```diff
@@ -5,160 +5,128 @@
...
-[**--runtime**=[*value*]]
-[**--seccomp-profile**=[*value*]]
-[**--selinux**]
-[**--signature-policy**=[*value*]]
-[**--storage-driver**=[*value*]]
-[**--storage-opt**=[*value*]]
-[**--version**|**-v**]
-
+```
+crio                               \
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md]: This is a nice change over all, but I'm on the fence about the '\' at the end of the line.  How does it look without it?

File: docs/crio.8.md [hunk 2]
```diff
@@ -5,160 +5,128 @@
...
-**crio [GLOBAL OPTIONS] config [OPTIONS]**
-
+- Support multiple image formats including the existing Docker image format
+- Support for multiple means to download images including trust & image verification
+- Container image management (managing image layers, overlay filesystems, etc)
+- Container process lifecycle management
+- Monitoring and logging required to satisfy the CRI
+- Resource isolation as required by the CRI
+
+Usage:
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md]: Can you make 'Usage:' be bold without increasing the font size?

File: docs/crio.8.md [hunk 3]
```diff
@@ -5,160 +5,128 @@
...
 
-**--help, -h**
-  Print usage statement
+- **--help, -h**: Print usage statement
 
-**--insecure-registry=**
-  Enable insecure registry  communication,  i.e.,  enable  un-encrypted
+- **--insecure-registry=**: Enable insecure registry  communication,  i.e.,  enable  un-encrypted
   and/or untrusted communication.
+  > List  of  insecure registries can contain an element with CIDR notation
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md]: This change  puts these lines in a new font.  Can you just make these a bullet list as you did image-volumes below?

**Comment by willstudy** [comment]
@TomSweeneyRedHat

**Comment by rhatdan** [comment]
The generated man page seems to have random "." in its output.

```
       --cpu-profile="": set the CPU profile file path

       ·

       --default-transport: A prefix to prepend to image names that can't be
       pulled as-is.

       ·

       --help, -h: Print usage statement

       ·
````

**Comment by rhatdan** [comment]
Move the crio in the SYNOPSYS block out of the ``` BLOCK ``` then it looks better and removing the bullets from the --options makes the man page look good.

 sed 's/^-//g' docs/crio.8.md | go-md2man -out /tmp/crio.8

**Comment by willstudy** [comment]
man page

**Comment by TomSweeneyRedHat** [comment]
LGTM and thanks!

**Comment by rhatdan** [comment]
LGTM
Thanks @willstudy

**Pull request merged**